app/context/extra_keyboard/index.tsx

```diff
@@ -182,7 +182,7 @@ const ExtraKeyboardComponent = () => {
     });
 
     const animatedStyle = useAnimatedStyle(() => {
-        let height = keyb.height.value + offset;
+        let height = keyb.height.value;
```

Comment:
Why this change?

Reply:
Ah, removed during testing something. Will revert it back.